helm-chart/renku-notebooks/values.yaml

```diff
@@ -10,11 +10,31 @@ global:
     domain:
   anonymousSessions:
     enabled: false
+  s3Datasets:
+    enabled: false
 
 amalthea:
   scope:
     clusterWide: false
   deployCrd: true
+  ## If enabling S3 datasets uncomment the section below
+  # extraChildResources:
+  #   - name: datasets
+  #     group: com.ie.ibm.hpsys
+
+## If enabling S3 datasets uncomment the section below
```

Comment:
I think these can be uncommented per default, if I'm not mistaken they will just be ignored if the datashim chart is not deployed.